### ISSUE TITLE
Expose timeout configuration for aws ebs snapshot

### ISSUE THREAD

**Issue by Ihar87** [terraform-providers/terraform-provider-aws#8043]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
When taking a snapshot of a volume I'm getting exceeded wait attempts:

```
[0m[1maws_ebs_snapshot.artifact_ebs_volume: Still creating... (9m1s elapsed)[0m[0m
[0m[1maws_ebs_snapshot.artifact_ebs_volume: Still creating... (9m11s elapsed)[0m[0m
[0m[1maws_ebs_snapshot.artifact_ebs_volume: Still creating... (9m21s elapsed)[0m[0m
[0m[1maws_ebs_snapshot.artifact_ebs_volume: Still creating... (9m31s elapsed)[0m[0m
[0m[1maws_ebs_snapshot.artifact_ebs_volume: Still creating... (9m41s elapsed)[0m[0m
[31m
[1m[31mError: [0m[0m[1mError applying plan:

1 error(s) occurred:

* aws_ebs_snapshot.artifact_ebs_volume: 1 error(s) occurred:

* aws_ebs_snapshot.artifact_ebs_volume: ResourceNotReady: exceeded wait attempts
```

### New or Affected Resource(s)

- aws_ebs_snapshot

- Terraform Version v0.11.7

* provider.aws: version = "~> 2.2"

### Potential Terraform Configuration

```hcl
resource "aws_ebs_snapshot" "artifact_ebs_volume" {
  volume_id   = "${var.volume_id}"

  tags = {
    Name = "${var.snap_name}"
  }
```

### Steps to Reproduce

- terraform apply

### Proposal

- Increase default timeout for the AWS provider or provide timeout configuration.

**Comment by bflad** [comment]
Increasing the default timeout for EBS Snapshot creation/deletion to 10 minutes and supporting customizable timeouts has been merged and will release with version 2.19.0 of the Terraform AWS Provider, likely later this week. Thanks, @res0nance!

**Comment by bflad** [comment]
This has been released in [version 2.19.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2190-july-11-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!
